src/ScriptCs.Core/ScriptLibraryWrapper.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using ScriptCs.Contracts;
+
+namespace ScriptCs
+{
+    public abstract class ScriptLibraryWrapper
```

Comment:
I am adding it to IScriptHost. This means moving ScriptEnvironment to Contracts, OR having an IScriptEnvironment in Contracts. As ScriptEnvironment is so light, I am thinking to just move it to contracts.